deblur/test/data/testmerge.fastq.trim.derep.no_artifacts.msa.deblur.no_chimeras

```diff
@@ -0,0 +1,20 @@
+>810205;size=1
```

Comment:
all deblurred sample files are fasta. the .fastq just means it started as a .fastq, but it turns into a fasta file after the .trim step